icons/freedesktop-dot-org.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>freedesktop.org icon</title><path d="M17.7027 1.9336a4.3775 4.3775 0 00-.3824.0033 4.4174 4.4174 0 00-.8822.1375L3.2994 5.5021C.945 6.1164-.4708 8.5314.1434 10.8858l2.0563 7.8834c.6143 2.3547 3.0299 3.7707 5.3843 3.1564l13.1387-3.4282c2.2126-.5773 3.5736-2.7456 3.2223-4.9588h.0447l-.111-.425-2.0564-7.8833c-.5039-1.9313-2.2196-3.231-4.1196-3.2967zm-.021.6774c1.6094.0534 3.0553 1.1507 3.4832 2.7908l1.9458 7.4578.111.425v.0006c.522 1.9993-.671 4.0337-2.6705 4.5553L7.4125 21.2682c-1.9994.5217-4.0342-.6708-4.5559-2.6705L.8003 10.7144C.2786 8.715 1.4714 6.6806 3.4709 6.159l13.1387-3.4277c.2499-.0653.5005-.1037.7485-.117a3.712 3.712 0 01.3236-.0033zM17.29 3.635c-.3753-.0015-.7756.044-1.1972.1386L4.4308 6.8164c-2.4282.7063-3.1071 2.0162-2.6242 4.2652l1.7495 6.7045c.4829 2.1136 2.2831 3.1013 4.373 2.4397l11.6622-3.0427c2.1228-.5042 3.1071-2.0966 2.6242-4.2652L20.466 6.2134c-.3924-1.6763-1.5495-2.5718-3.1759-2.5783zm1.2512 3.5528c.2161-.0126.413.1263.4703.3457l.6228 2.388v.0005l.0108.0421c.0654.2502-.0796.4981-.33.5636l-3.4046.8882c-.2508.0653-.4986-.0799-.564-.3306l-.6337-2.43c-.0655-.2508.0795-.4986.3306-.5641l3.4045-.8882a.4794.4794 0 01.0933-.0151zM8.325 7.847c.3002-.017.575.1763.6542.48l.777 2.978c.0906.3467-.113.6935-.46.7841l-4.1724 1.0888c-.3473.0906-.6938-.1126-.7842-.4594l-.777-2.9785a.6377.6377 0 01.4594-.7842l4.173-1.0883a.6654.6654 0 01.13-.0204zm5.8578 1.4664l.173.6633-4.1206.4751-.1732-.6633zm.8995 2.5406c.184.147.4105.234.6498.248L14.158 14.471c-.1354-.1347-.3108-.226-.501-.2545a.8718.8718 0 00-.1402-.007zm-5.627.8952l2.0315 1.9608-.5193.1354h-.0005c-.0796.0208-.1516.055-.2195.095l-2.0623-1.99zm4.1062 2.1377c.0816.0109.15.0688.1726.1559v.0005l.4244 1.6265.0108.0421c.0304.1165-.034.2274-.1505.2578l-2.3378.6094c-.1163.0303-.2268-.0342-.2572-.151l-.4358-1.6686c-.0303-.1163.0343-.2267.151-.2572l2.3378-.61a.2189.2189 0 01.0847-.0054z"/></svg>
```

Comment:
There are some ***very*** slight differences between the PR path (red) and source at 7 decimals (black).
![image](https://user-images.githubusercontent.com/1830867/102896056-b25cb700-445d-11eb-891c-0f7f91a8825d.png)

Could you try and rebuild to 3 decimals again, see if it changes anything?
If not, @simple-icons/maintainers are we okay with these *slight* differences?

Reply:
I think as long as they can be avoided at a [precision of 5 or less](https://github.com/simple-icons/simple-icons/blob/5e4b4d66672208daceb77bf077fdfa03afa58e4b/.svglintrc.js#L14) any differences with an official vector version should be avoided.

Reply:
I can see the same slight diffferences. After some messing around with the icon a bit, it seems like it's Inkscape's fault, converting some strokes to paths that is causing this.

Unless someone else wants to give it a try with Illustrator etc, we probably have to stick with it.

Reply:
> Unless someone else wants to give it a try with Illustrator etc, we probably have to stick with it.

I can take a quick look in Illustrator once I've ran through the backlog of updated icons 😅

Reply:
Can another @simple-icons/maintainers please review this with the latest push?

Reply:
@adamrusted Looks good to me! Better than my version. 💯